components/LessonTitleCard.tsx

```diff
@@ -24,7 +27,18 @@ const LessonTitleCard: React.FC<LessonTitleProps> = props => {
           />
           <div>
             <p className="m-0">
-              <NavLink path="/curriculum">Go Back</NavLink>
+              <a
+                href=""
```

Comment:
Use `href="#"` so it doesn't actually link to anything.

```suggestion
                href="#"
```